modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,295 @@
+
+----
+(undercloud) $ sudo openstack overcloud container image upload \
+  --config-file  /home/stack/local_registry_images.yaml \
+  --verbose
+----
+
+
+As an Octavia Load Balancer is used to access the OpenShift API, there is a
+need to increase their listeners default timeouts for the connections.
+The default timeout is 50 seconds. Increase the timeout to 20 minutes by
+passying the following file to the overcloud deploy command:
+
+----
+(undercloud) $ cat octavia_timeouts.yaml
+parameter_defaults:
+  OctaviaTimeoutClientData: 1200000
+  OctaviaTimeoutMemberData: 1200000
+----
+
+[NOTE]
+This is not needed from Red Hat OpenStack Platform 14 and onwards.
+
+Install or update your overcloud environment with Octavia:
+
+----
+openstack overcloud deploy --templates \
+.
+.
+.
+  -e /usr/share/openstack-tripleo-heat-templates/environments/services-docker/octavia.yaml \
+  -e octavia_timeouts.yaml
+.
+.
+.
+----
+
+[NOTE]
+The command above only includes the files associated with Octavia. This command
+will vary based upon your specifc installation of OpenStack. See the official
+OpenStack documentation for further information. For more information on
+customizing your Octavia installation, see
+https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/networking_guide/#planning_your_octavia_deployment[installation
+of Octavia using Director].
+
+[NOTE]
+If Kuryr SDN is used, the overcloud installation requires the "trunk" extension
+to be enabled at Neutron. This is enabled by default on Director deployments.
+Use the openvswitch firewall instead of the default ovs-hybrid when the Neutron
+backend is ML2/OVS. There is no need for modifications if the backend is
+ML2/OVN.
+
+In order to be able to enforce Network Policies across services (i.e., when
+traffic goes through the Octavia load balancer), we need to ensure Octavia
+creates the amphora VM security groups on the user project. To do that, you
+must add the project ID to the `octavia.conf` configuration file after you
+create the project. This step ensures that required LoadBalancer security
+groups belong to that project and that they can be updated to enforce services
+isolation.
+
+. Get the project ID
++
+----
+$ openstack project show *<project>*
++-------------+----------------------------------+
+| Field       | Value                            |
++-------------+----------------------------------+
+| description |                                  |
+| domain_id   | default                          |
+| enabled     | True                             |
+| id          | PROJECT_ID                       |
+| is_domain   | False                            |
+| name        | *<project>*                      |
+| parent_id   | default                          |
+| tags        | []                               |
++-------------+----------------------------------+
+----
++
+. Add the project ID to `octavia.conf` on the controllers and restart octavia
```

Comment:
s/on the controllers and restart octavia/for the controllers and restart the Octavia